README.md

```diff
@@ -26,17 +27,17 @@ The UppHub global registry file, `nests.json`, is a  simple JSON file containing
 | `repository`  | The link to the nest's repository.                                  |
 | `status`      | The status of the nest: `stable`, `experimental`, etc.              |
 | `category`    | The category of the nest: `networking`, `scientific`, `widget`, etc.|
-| `readme`      | A link to the `README` file. This entry is optional.                |
+| `readme`      | A link to the `README` file.                                        |
```

Comment:
README is optional - nothing will happen if it will not be present? Do we want to change this policy? On TheIDE side the README button will not be present.

Reply:
@klugier 
It is already stated in the notes.